### ISSUE TITLE
Update to oemof version

### ISSUE THREAD

**Issue by SabineHaas** [rl-institut/oemof-B3#193]
Once oemof.tabular is updated to the new oemof version we can require a newer oemof version.

Once the equate  flows constraint is merged to dev we can require this version containing [equate_flows.py](https://github.com/oemof/oemof-solph/blob/features/equate-flows/src/oemof/solph/constraints/equate_flows.py)   ( or require branch `features/equate-flows`).

Then `oemof_b3/tools/equate_flows.py` can be deleted.

**Comment by jnnr** [comment]
oemof-B3 now runs with oemof.solph v0.4. Next steps:
* Finalize https://github.com/oemof/oemof-solph/pull/817 and merge into oemof.solph v0.5
* Wait for oemof.solph v0.5 release
* Run oemof.tabular and oemoflex on oemof.solph v0.5
* Delete oemof_b3/tools/equate_flows.py
